Aquality.Selenium.Core/src/Aquality.Selenium.Core/Applications/Startup.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Aquality.Selenium.Core.Configurations;
+using Aquality.Selenium.Core.Elements;
+using Aquality.Selenium.Core.Elements.Interfaces;
+using Aquality.Selenium.Core.Localization;
+using Aquality.Selenium.Core.Logging;
+using Aquality.Selenium.Core.Utilities;
+using Aquality.Selenium.Core.Waitings;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Reflection;
+
+namespace Aquality.Selenium.Core.Applications
+{
+    /// <summary>
+    /// Allows to resolve dependencies for all services in the Aquality.Selenium.Core library
+    /// </summary>
+    public class Startup
+    {
+        /// <summary>
+        /// Used to configure dependencies for services of the current library
+        /// </summary>
+        /// <param name="services">Collection of service descriptors</param>
+        /// <param name="applicationProvider">function that provides an instance of <see cref="IApplication"/></param>
+        /// <param name="settingsFile"><see cref="JsonFile"/> with settings for configuration of dependencies.
+        /// Pass the result of <see cref="GetSettings"/> if you need to get settings from the embedded resource of your project.</param>
+        public void ConfigureServices(IServiceCollection services, Func<IServiceProvider, IApplication> applicationProvider, JsonFile settingsFile = null)
```

Comment:
Maybe would be better to switch from JsonFile to ISettingsFile interface or something like this?

Reply:
Probably yes. Should we have interfaces for everything?